main.js

```diff
@@ -1,9 +1,75 @@
-var images = ['candle1', 'candle2', 'candle3', 'candle4', 'candle5', 'candle6']
+var candles = [
+  {
+    id: 0,
+    name: 'Peace Becomes Her',
+    description: 'Fend off death with this peaceful mixture of aged juniper and winter cranberry. Burns for 100 hours. Wood wick.',
+    source: 'http://www.yankeecandle.com/media/images/product/medium/1351702.jpg'
+  },
+  {
+    id: 1,
+    name: 'Serentiy, NOW!',
+    description: 'Don\'t beat around the bush, get serene NOW. Peppermint and basil medle together for ahh..serenity. Burns for 100 hours. Wood wick.',
+    source: 'http://www.yankeecandle.com/media/images/product/medium/1351702.jpg'
+  },
+  {
+    id: 2,
+    name: 'Shangri-lovely',
+    description: 'Madarin and vanilla channel the scents of Shangria-La, whisking you away to the jungle. Burns for 100 hours. Wood wick',
+    source: 'http://www.yankeecandle.com/media/images/product/medium/1073481.jpg'
+  },
+  {
+    id: 3,
+    name: 'Aloahhhh',
+    description: 'Feel like your beachside in Hawaii with coconut and sugar filling the air. Burns for 110 hours. Wood wick.',
+    source: 'http://www.yankeecandle.com/media/images/product/medium/1073481.jpg'
+  },
+  {
+    id: 4,
+    name: 'The Office',
+    description: 'Maybe you\'re most relaxed at work. This clean, sterile mixture of mint and cotton will bring you back to your desk, feeling productive. Burns for 100 hours. Wood wick.',
+    source: 'http://www.yankeecandle.com/media/images/product/medium/1351702.jpg'
+  },
+  {
+    id: 5,
+    name: 'Meadow Spring',
+    description: 'Bambi was rarely stressed. Feel like a fawn in Spring with this delicious lavander and orange mixture. Burns for 110 hours. Wood wick.',
+    source: 'http://www.yankeecandle.com/media/images/product/medium/1351702.jpg'
+  },
+]
+
+for(i = 0; i<candles.length; i++) {
+  var $candleContainer = document.createElement('div')
```

Comment:
You should separate your DOM creation into its own function and use it in your loop.

```js
var $gallery = document.querySelector('#gallery')

for (var i = 0; i < candles.length; i++) {
  $gallery.appendChild(renderCandle(candles[i]))
}
```